### PR TITLE
snackbar&rating

### PR DESCRIPTION
**Pull request by preethasri**
created snackbar and rating component
created live demo for snackbar and rating

### CODE REVIEW

**Review by manojkumar8400 — APPROVED** [review]
great work.

File: ratings/rating.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+* {
+    font-family: 'Nunito Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    --rating-color: #FFDF00;
+}
+
```
Comment (mimansha-swarup) [inline ratings/rating.css:8]: It look like you are repeating code of documentation page
why not keep it in separate file and import directly to HTML file

File: ratings/rating.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+.side-heading {
+    font-size: 1.4rem;
+    font-weight: bold;
+    margin-bottom: 1.5rem;
+}
+
+.ratings {
+    background-color: #f1f1f1;
```
Comment (mimansha-swarup) [inline ratings/rating.css:27]: use variable

File: snackbar/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    display: flex;
+    justify-content: center;
+    margin: 20px 0px;
+}
+
+.toast-demo {
+    display: flex;
+    justify-content: flex-start;
+    gap: 10px;
+    background-color: #f1f1f1;
```
Comment (mimansha-swarup) [inline snackbar/snackbar.css:40]: use variable

File: snackbar/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    padding: 15px 15px;
```
Comment (mimansha-swarup) [inline snackbar/snackbar.css:41]: using rem or em will be better

File: snackbar/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+.live-demo-button {
+    background-color: #f1f1f1;
+    padding: 15px 15px 15px 15px;
+    border: none;
+    margin-bottom: 100px;
+}
+
+.live-demo-button:hover {
+    background-color: lightblue;
```
Comment (mimansha-swarup) [inline snackbar/snackbar.css:82]: use variable

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: ratings/rating.css (hunk 1 above)
Comment (preethasri) [reply ratings/rating.css:8]: hey mimansha  thank you for pointing out. i created the common file for it .once i merge this i will remove this thing

File: snackbar/snackbar.css (hunk 5 above)
Comment (preethasri) [reply snackbar/snackbar.css:82]: sure will work on this👍

**Comment by preethasri** [comment]
> great work.

Thanks Manoj

**Review by vijaytembugade — APPROVED** [review]
Looks good to me.

**Comment by preethasri** [comment]
> Looks good to me.

thanks vijay

**Pull request merged**